src/plugins/profiling/server/routes/search_flamechart.ts

```diff
@@ -70,64 +77,80 @@ export function registerFlameChartSearchRoute(router: IRouter<DataRequestHandler
       },
     },
     async (context, request, response) => {
-      const { index, projectID, timeFrom, timeTo } = request.query;
-      const sampleSize = 200;
+      const { projectID, timeFrom, timeTo } = request.query;
+      const targetSampleSize = 20000; // minimum number of samples to get statistically sound results
+      const topN = 200; // collect data for the top N unique stack traces
 
       try {
         const esClient = context.core.elasticsearch.client.asCurrentUser;
         const filter = newProjectTimeQuery(projectID!, timeFrom!, timeTo!);
 
-        // const resp = await getCountResponse(context, filter);
+        // Start with counting the results in the index down-sampled by 5^6.
+        // That is in the middle of our down-sampled indexes.
+        const initialExp = 6;
         const resp = await esClient.search({
-          index: 'profiling-events-1000',
+          index: 'profiling-events-5pow' + initialExp,
           body: {
             query: filter,
             size: 0,
             track_total_hits: true,
           },
         });
+        const sampleCountFromInitialExp = resp.body.hits.total.value as number;
 
-        const sampleCountFromSmallestTable = resp.body.hits.total.value as number;
+        console.log('sampleCountFromPow6', sampleCountFromInitialExp);
 
-        const [eventsIndex, sampleRate, estimatedSampleCount] = getSampledTraceEventsIndex(
-          sampleSize,
-          sampleCountFromSmallestTable
+        const eventsIndex = getSampledTraceEventsIndex(
+          targetSampleSize,
+          sampleCountFromInitialExp,
+          initialExp
         );
 
         // eslint-disable-next-line no-console
-        console.log('Index', eventsIndex, sampleRate, estimatedSampleCount);
+        console.log('EventsIndex', eventsIndex);
```

Comment:
Should we find these logs in the web console? I think nope since they run in the nodejs process of Kibana itself.
If that's the case, are they useful?